README.md

```diff
@@ -123,26 +139,6 @@ In the other window, run this command - This can automatically trigger the step
 yarn workspace comms-app-react-native-videocall android
 ```
 
-## Features
```

Comment:
Why remove? It was feedback from QA that he doesn't know what a user shall expect in the app.

Reply:
Moved into overview above, the table :)